stl/src/filesys.cpp

```diff
@@ -459,22 +459,20 @@ _FS_DLL int __CLRCALL_PURE_OR_CDECL _Copy_file(const wchar_t* _Fname1, const wch
 
 _FS_DLL int __CLRCALL_PURE_OR_CDECL _Chmod(const wchar_t* _Fname, perms _Newmode) {
     // change file mode to _Newmode
-    WIN32_FILE_ATTRIBUTE_DATA _Data;
-
-    if (!GetFileAttributesExW(_Fname, GetFileExInfoStandard, &_Data)) {
+    DWORD _Oldmode = GetFileAttributesW(_Fname);

```

Comment:
This would:
 * potentially will improve performance by doing more narrow request (could be a worth enhancement)
 * save stack space (a very low priority enhancement)
 * might enable some older platforms (a non-goal)

Could be worth doing to the current filesystem, but not worth changing the legacy experimental one